code/script.js

```diff
@@ -0,0 +1,242 @@
+const username = 'katiewu1';
+const API_REPOS = `https://api.github.com/users/${username}/repos`;
+const API_USER = `https://api.github.com/users/${username}`;
+
+const projectContainer = document.getElementById('projectContainer');
+const profile = document.getElementById('profile');
+const modalContent = document.getElementById('modalContent');
+const closeBtn = document.getElementById('closeBtn');
+const modalProject = document.getElementById('modalProject');
+
+// function to get the repositories
+const getRepos = () => {
+  fetch(API_REPOS)
+    .then((response) => {
+      return response.json();
+    })
+    .then((json) => {
+      // filter out the repos that have been forked
+      // in 'json[index].fork' we can find the boolean (true/false)
+      // we want all the repo with boolean true
+
+      // we also want to filter and find the projects from Technigo
+      // the repos from Technigo always start with 'project-'
+      const filteredTechnigoRepos = json.filter(
+        (repo) => repo.fork && repo.name.includes('project-')
+      );
+
+      // store the project names in a new array called namesOfTheRepos
+      const namesOfTheRepos = filteredTechnigoRepos.map((repo) => repo.name);
+
+      // for each project, create a button element and class project-card
+      // inside each card display the project name and a emoji
+      // with append we push each project card
+      // and add an eventListener so we can click on each card and get a modal(popup)
```

Comment:
Could it be an idea to also have a direct link to github-repo in the first page? So the user can either "read more" 
on the pop-up window, or go "straight to github-repo".

Reply:
It's possible. But have to do another container for that I think 🤔  Because I made the whole project-card clickable to get the pop up window